docs/website/docs/reference/tuning.md

```diff
@@ -0,0 +1,126 @@
+---
+icon: octicons/meter-16
+---
+
+# Tuning
+
+This page documents support for IREE dispatch tuning. The compiler supports
+both default and user-provided tuning specs that override compiler heuristics
```

Comment:
Relating to my other comment about abbreviations, maybe spell out "specification" instead of "spec"? I don't feel too strongly.

Reply:
This is consistent with the op naming in transform dialect, so maybe I'll expand this ones in parens and leave the rest as-is